eqcorrscan/utils/archive_read.py

```diff
@@ -216,6 +231,11 @@ def _check_available_data(archive, arc_type, day):
                 for channel in station:
                     available_stations.append((station.code,
                                                channel.code))
+    elif arc_type.lower() == 'sds':
```

Comment:
Just for consistency: 
```suggestion
    elif arc_type.upper() == "SDS":
```